daemon/c_fifo.c

```diff
@@ -168,12 +235,22 @@ c_fifo_loop(char *fifo_path_c0, char *fifo_path_container)
 			TRACE_ERRNO("Failed to close old reading fd");
 		}
 
+		if (!file_is_fifo(fifo_path_c0)) {
+			ERROR("Could not open FIFO at %s, stopping forwarder", fifo_path_c0);
+			exit(EXIT_SUCCESS);
+		}
+
 		if (-1 == (fromfd = open(fifo_path_c0, O_RDONLY))) {
 			ERROR_ERRNO("Failed to open fromfd at %s, exiting...", fifo_path_c0);
 			exit(-1);
 		}
 		TRACE("Opened reading end for %s", fifo_path_c0);
 
+		if (!file_is_fifo(fifo_path_container)) {
+			ERROR("Could not open FIFO at %s, stopping forwarder", fifo_path_c0);
```

Comment:
shouldnt fifo_path_c0 be replaced with fifo_path_container here?

Reply:
changed it, thnx for spotting